data/campaigns/Eastern_Invasion/scenarios/04a_An_Elven_Interlude.cfg

```diff
@@ -216,6 +216,7 @@
         [/message]
         [message]
             speaker=Vaelya
+            #po: followed by a delay and then the elves switching from allies to enemies
```

Comment:
Not relevant for translation

The question you might ask yourself: if any information revealed in the po hint was different, would the meaning of the string change?

Reply:
I'm thinking there could be a bit of change to the language depending on how quickly the elves turn to enemies, or whether they would haggle for gold first.

Reply:
I also find it hard to see how this would be relevant to the translation. I don't want to say it definitely won't be though, since I don't know every language.